client/go/dogma/dogma.go

```diff
@@ -77,16 +78,20 @@ type service struct {
 	client *Client
 }
 
-// NewClient returns a Central Dogma client with the specified baseURL, clientID and clientSecret.
-func NewClient(baseURL string, clientID, clientSecret string) (*Client, error) {
+// NewClient returns a Central Dogma client with the specified baseURL, username and password.
+func NewClient(baseURL string, username, password string) (*Client, error) {
```

Comment:
nit: Just curious. `username` -> `username string`? Is the type unnecessary?

Reply:
Oops~! Thank you for pointing it out. `(baseURL, username, password string)` is correct because all of the three parameters are string, I can just put one `string` at the end. (does not have to put type for every parameter, if they are same)